mods/cnc/maps/nod05/map.yaml

```diff
@@ -342,7 +342,7 @@ Actors:
 	Tower2: gtwr
 		Location: 23,21
 		Owner: GDI
-	Radar: hq
```

Comment:
TD Radar is not a Radar Dome, it is called Communitcaitions Center, CommCenter should do for unit's name.